user_manual/files/encrypting_files.rst

```diff
@@ -57,9 +57,8 @@ If you don't have the users password or `file recovery key
 
 .. TODO ON RELEASE: Update version number above on release
 
-then there is no way to decrypt all files. What's
-more, running it on login would be dangerous, because you would most likely run
-into timeouts.
+Then there is no way to decrypt all files. What's more, running it on login
```

Comment:
But this continues the sentence from above? `If you don't have the users password or `file recovery key …`